include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -1588,6 +1588,223 @@ namespace internal
           *local_values_begin = read_write_vector[sorted_indices_pos[i]];
       }
 #endif
+
+
+
+      template <typename DoFHandlerType,
```

Comment:
Can you add a short comment here, especially regarding the `Operation` type. While it is used elsewhere, it takes a little to grasp it, so let's be explicit.